app/index.html

```diff
@@ -15,6 +15,15 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700' rel='stylesheet' type='text/css'>
 
     {{content-for "head-footer"}}
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
```

Comment:
Discussed with developer: there's a fair chance that we don't need MathML, but the file size is small enough to be manageable either way. 

We'll pare down in the future if performance becomes an issue later.